### PR TITLE
allows user to update votes for each article

### PR DESCRIPTION
**Pull request by harrys1000rr**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -72,6 +70,30 @@ describe('GET - /api/articles/:article_id', () => {
...
+      .then((res) => {
+        expect(res.body.article).toEqual(
+          expect.objectContaining({
+            article_id: 2,
+            title: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number)
```
Comment (itskatherine) [inline __tests__/api.test.js]: Will need here to check that the vote count has been increased by the amount specified in the pull request. For example, if it was 50 before, you'd expect it to be 100 after the patch 🙂

File: __tests__/api.test.js [hunk 2]
```diff
@@ -72,6 +70,30 @@ describe('GET - /api/articles/:article_id', () => {
   });
 })
```
Comment (itskatherine) [inline __tests__/api.test.js]: Need a few more tests and error handling for this patch request, we'll need to check
- 400 invalid article id
- 404 valid article id but doesn't exist
- 400 missing or incorrect body (like if inc_votes is missing or incorrect)

**Review by itskatherine — COMMENTED** [review]
A few more tests and some more error handling needed, see comments for more deets, but a good start 🙂 
Could you please send another pull request for this ticket once you've added these changes
👍

**Comment by harrys1000rr** [comment]
Duplicate pull request

**Pull request closed without merging**